src/psyclone/generator.py

```diff
@@ -99,15 +99,14 @@ def handle_script(script_name, info, function_name, is_optional=False):
     '''
     sys_path_appended = False
     try:
-        # a script has been provided
+        # a script file has been provided, check it exists
+        if not os.path.isfile(script_name) and not any(
+               os.path.isfile(os.path.join(p, script_name)) for p in sys.path):
```

Comment:
To answer my own question, I see that the code has always behaved this way (and I've either forgotten or never noticed). In which case I'm happy (although is there a utility that searches the `sys.path` so that we don't have to?).

Reply:
I've rewritten `handle_script()` so you no longer need the `./`.
`__import__()` reads the system path, so that's why we check it.
I'm not aware of any such utility.